packages/@aws-cdk/region-info/README.md

```diff
@@ -0,0 +1,80 @@
+# AWS Region-Specific Information Directory
+## Usage
+Some information used in CDK Applications differs from one AWS region to
+another, such as service principals used in IAM policies, S3 static website
+endpoints, ...
+
+### Direct Access
+This library offers a primitive database of such information so that CDK
+constructs can easily access regional information. The `Facts` class provides a
+list of known fact names, which can then be used with the `RegionInfo` to
+retrieve a particular value:
+
+```ts
+import { Facts, RegionInfo } from '@aws-cdk/region-info';
+
+const codeDeployPrincipal = RegionInfo.find('us-east-1', Facts.servicePrincipal('codedeploy.amazonaws.com'));
+// => codedeploy.us-east-1.amazonaws.com
+
+const staticWebsite = RegionInfo.find('ap-northeast-1', Facts.s3StaticWebsiteEndpoint);
+// => s3-website-ap-northeast-1.amazonaws.com
+```
+
+### Token Access
```

Comment:
Since we can represent tokens as strings, why would users even care about this? I would argue that they should always get a `string` with a stringified token as a response (so they are not tempted to read the value). I don't see value in exposing a `Token` class here.

Reply:
There's a lot of usage sites for this information in contexts that are NOT bound to a Construct. Where do you get the region from?